### ISSUE TITLE
--help/--version should support command format

### ISSUE THREAD

**Issue by dlo9** [iocage/iocage#972]
The options --help and --version are currently confusing and (for me at least) it's a common mistake to type `iocage help` instead of `iocage --help` given that other iocage arguments (the 'commands') are not preceded with the standard argument prefix `--`. Given that the intent of `iocage help` is clear and unambiguous, it would make sense and improve the tool's ergonomics to support this usage style.

- [x] Supply `iocage --version`
`Version 1.2 RC` (`master` at the time of this issue's creation, commit 812c7b5b880581bd7e6ed9e67fb120048ec89492)
- [x] Supply the commands used, along with any steps to recreate it.
`iocage help`
- [x] Provide the output from the command you issued.
```
  File "./iocage", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1132, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1171, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/root/iocage/iocage_cli/__init__.py", line 182, in get_command
    mod = __import__(f"iocage_cli.{name}", None, None, ["cli"])                                                                                                                             
ModuleNotFoundError: No module named 'iocage_cli.help'
```
- [x] Supply what you expected the result or output to be
The same output as `iocage --help`, e.g. the output of help when used with no commands. 
- [x] Checked that the problem has not already been fixed on `master` if using
 a stable release.

**Comment by jsegaert** [comment]
The traceback was caused because it wasn't properly validating the _command_.  That was fixed via pull request #979.   The latest master now behaves as below:
```
➜  ~ iocage help
Usage: iocage [OPTIONS] COMMAND [ARGS]...
Try "iocage --help" for help.

Error: No such command "help".
➜  ~  
```

**Comment by dlo9** [comment]
Thanks! Has there been any consideration of the suggestion to make `help` a command? Many other tools take this route to simplify the tool's usage (e.g. `git help` and `git --help` behave the same).
